tests/app/views/contexts/test_calculated_summary_context.py

```diff
@@ -94,7 +94,7 @@ def test_build_view_context_for_currency_calculated_summary(
 
     assert "summary" in context
     assert_summary_context(context)
-    assert len(context["summary"]) == 6
+    assert len(context["summary"]) == 7
```

Comment:
The content of `headers` should be unit tested. I don't see one where answers aren't editable.